lib/dicthf.gi

```diff
@@ -40,7 +41,11 @@ InstallMethod(SparseIntKey,"defaults to DenseIntKey",true,[IsObject,IsObject],
 ##  between different runs of {\GAP} and no reference to their absolute
 ##  values ought to be made.
 ##
+##  HashKeyWholeBag hashes all the contents of a bag, which is equivalent
```

Comment:
Should the documentation warn the user that many types of Bag have dirty free space of one kind or another at the end.  Alternatively, we should launch a systematic campaign to get rid of such.